ui/tests/integration/components/clients/page-header-test.js

```diff
@@ -5,7 +5,7 @@
 
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'vault/tests/helpers';
-import { click, fillIn, render } from '@ember/test-helpers';
+import { click, fillIn, render, waitFor, waitUntil } from '@ember/test-helpers';
```

Comment:
I opted to use these helpers instead of adding helpers like `@waitForPromise` to the `rawRequest` functions in production code. Reason being testing helpers should really be avoided in production, if possible! https://www.npmjs.com/package/@ember/test-waiters